test/users-test.js

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { sampleUsersData } from '../src/data/sample-users';
+import { getRandomUser } from '../src/users'
+
+describe('select a random user', () => {
```

Comment:
this can be combined in the getRandomRecipe() tests to check that the function can take in a different list